Sources/Client/Model/Channel/Channel.swift

```diff
@@ -138,6 +140,7 @@ public final class Channel: Codable {
     
     public init(type: ChannelType,
                 id: String,
+                membership: Member? = nil,
```

Comment:
Is this something clients can configure?

Reply:
I don't think so. I think I should remove it